lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -166,7 +172,13 @@ export default class BitcoinProcessor {
     }
 
     console.info(`Starting block: ${startingBlock.height} (${startingBlock.hash})`);
-    await this.processTransactions(startingBlock);
+    // this reads into the raw block files and parse to speed to the initial startup
+    // will use this instead of the rpc processing when ready
+    if (this.bitcoinDataDirectory) {
+      await this.fastProcessTransactions(startingBlock);
+    } else {
+      await this.processTransactions(startingBlock);
```

Comment:
Is it possible to add the equivalent code for fetching the coinbase transaction data in `processTransaction` also?

Reply:
Yes. We currently throw the the coinbase script away because we don't need it